src/blob.rs

```diff
@@ -915,4 +919,49 @@ mod tests {
         let avatar_cfg = t.get_config(Config::Selfavatar).await.unwrap();
         assert_eq!(avatar_cfg, avatar_blob.to_str().map(|s| s.to_string()));
     }
+
+    #[async_std::test]
+    async fn test_media_quality() -> anyhow::Result<()> {
+        for (media_quality_config, image_size) in vec![
+            (Some("0"), 1000), // BALANCED_IMAGE_SIZE > 1000, the original image size, so the image is not scaled down
```

Comment:
in a subsequent pr, consider adding an image with eg. 2000x2000 px, so that scaling can be testing in more variations (if the jpg only contains few information, even these huge images won't blow up the repo too much)